packages/server/api/test/integration/ce/flows/flow-step-test-output.test.ts

```diff
@@ -22,69 +22,58 @@ afterAll(async () => {
 });
 
 describe('Flow Step Test output', () => {
```

Comment:
i extracted common logic so that the individual test cases wouldnt be so long haha